odoo/orm/models.py

```diff
@@ -4482,7 +4480,7 @@ def _add_precomputed_values(self, vals_list: list[ValuesType]) -> None:
         """ Add missing precomputed fields to ``vals_list`` values.
         Only applies for precompute=True fields.
 
-        :param dict vals_list: list(dict) of create values
+        :param list[ValuesType] vals_list: list(dict) of create values
```

Comment:
No need to copy what is already in the signature.
```suggestion
        :param vals_list: list(dict) of create values
```